reconcile/utils/openshift_resource.py

```diff
@@ -237,6 +242,14 @@ def has_valid_sha256sum(self):
         except KeyError:
             return False
 
+    def has_valid_expiration_date(self):
+        try:
+            current_expiration_date = \
+                self.body['expirationDate']
+            return current_expiration_date == self.expiration_date()
+        except KeyError:
+            return False
+
```

Comment:
i can't find usage of this method anywhere besides the test you wrote for it itself. from the name of the method, i expect also something like checking if the date adheres to the format